saleor/graphql/account/types.py

```diff
@@ -440,6 +465,18 @@ def resolve_permission_groups(root: models.User, _info: ResolveInfo):
     def resolve_editable_groups(root: models.User, _info: ResolveInfo):
         return get_groups_which_user_can_manage(root)
 
+    @staticmethod
+    def resolve_accessible_channels(root: models.Group, info: ResolveInfo):
+        # Sum of channels from all user groups. If at least one group has
```

Comment:
Question: Wouldn't be good to use this part to extend the field description?

Reply:
Done